examples/aws-provider/composite-resources/backup/backup.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+apiVersion: awsblueprints.io/v1alpha1
+kind: Backup
+metadata:
+  name: backup
+  namespace: default
+spec:
+  compositionSelector:
+    matchLabels:
+      awsblueprints.io/provider: aws
+      awsblueprints.io/environment: dev
+  accountId: "123456789"
+  eksOIDC: "oidc.eks.eu-west-1.amazonaws.com/id/123456789A446538BDEA5AE78F3A68"
+  permissionsBoundaryArn: "arn:aws:iam::123456789:policy/crossplaneBoundary"
```

Comment:
Trying to get new examples to use EnvironmentConfig for such values. Can you do something similar to this example: https://github.com/awslabs/crossplane-on-eks/blob/e3893764d305aaa583e7618282668239d039afa6/examples/upbound-aws-provider/composite-resources/s3-irsa-example/README.md?plain=1#L43-L80